readme.md

```diff
@@ -38,63 +38,63 @@ $ npm -v
 
 * **Question:** What is the difference between forking and cloning a repository as opposed to just cloning a repository?
 
-* **Your Answer:**
+* **Your Answer:**  Fork copies to your own repo in Git.  Clone creating a local copy on your dev side.
 
 ---
 
 - [ ] Run `npm init -y` from the command line
 
 * **Question:** What does `npm init` do? How does the `-y` flag modify that command?
 
-* **Your Answer:**
+* **Your Answer:**  npm init creates the structure of the project.  -y is going to accept defaults, and use "yes".
 
 ---
 
 - [ ] Take a look at the file that was generated by the previous command
 
 * **Question:** What is the purpose of the following keys? "name", "scripts", "license"
 
-* **Your Answer:**
+* **Your Answer:**  name - project name.  scripts - what is going to be called.  license - licenses you have used in your project.
 
 ---
 
 - [ ] Create a `.gitignore` file
 
 * **Question:** What is the purpose of the `.gitignore` file? What is the significance of a "dot-file?"
 
-* **Your Answer:**
+* **Your Answer:** Files you do not want tracked by Git.
 
 ---
 
 - [ ] Create an `index.js` file with the following contents: `console.log('Hello, Node!')`
 
 * **Question:** From the command line, how can you run this file?
 
-* **Your Answer:**
+* **Your Answer:**  npm index.js
```

Comment:
It'll actually be `node index.js`. `node` is the program that allows to run JavaScript on our machine. `npm` is a package manager and gives us access to packages and scripts, among other things.